README.md

```diff
@@ -40,7 +40,7 @@ jobs:
     steps:
       - uses: actions/checkout@v2.2.0
         with:
-          fetch-depth: 0 # Required due to the weg Git works, without it this action won't be able to find any or the correct tags
+          fetch-depth: 0 # Required due to the way Git works, without it this action won't be able to find any or the correct tags
```

Comment:
The word `weg` in Dutch means `way`/`road/`street`/`gone`. Left it in to see how long it would take until someone would notice, turns out that is 7 months. Well done :+1:!